doc/client_lua_api.md

```diff
@@ -782,6 +782,41 @@ An interface to manipulate minimap on client UI
 * `set_shape(shape)`: Sets the minimap shape. (0 = square, 1 = round)
 * `get_shape()`: Gets the minimap shape. (0 = square, 1 = round)
 
+### Camera
+An interface to get or set information about the camera and cameranode.
+Please do not try to access the reference until a second or so after the client
+is fully loaded. The reference will be nil.
+
+#### Methods
+* `set_camera_mode(mode)`
+    * Pass `0` for first-person, `1` for third person, and `2` for third person front
+* `get_camera_mode()`
+    * Returns with same syntax as above
+* `get_fov()`
+    * Returns:
+    
+```lua
+    {
+        x = number,
+	y = number,
+	actual = number,
+	max = number
```

Comment:
Indents should be like here: https://github.com/minetest/minetest/pull/5609/files#diff-927bb329aa13096bcf9123248a422bc2R864